tests/test_db.py

```diff
@@ -0,0 +1,118 @@
+from asyncio import AbstractEventLoop
+from typing import List
+
+import pytest
+import testing.postgresql
+from asyncpg import Record
+
+from parking.backend.db.dbaccess import DbAccess
+from parking.shared.location import Location
+from parking.shared.rest_models import ParkingLot
+
+
+@pytest.mark.asyncio
+async def test_empty_parking_lots(event_loop: AbstractEventLoop):
```

Comment:
There should probably be an equivalent test for checking `create` creates an empty allocation table, too.